### ISSUE TITLE
inspec exec format json backtrace

### ISSUE THREAD

**Issue by alexpop** [inspec/inspec#614]
I've executed the following control:

``` ruby
control "one-control" do
  impact 1.0
  describe mount("/tmp") do
    it { should be_mounted  }
  end
end
```

with the `--format json` options and received the following output: [output.json.txt](https://github.com/chef/inspec/files/204558/output.json.txt)

This is the character count of the content with and without the backtrace content:

```
$ wc -m /tmp/output_w_backtrace.json
    4981 /tmp/output_w_backtrace.json

$ wc -m /tmp/output_wo_backtrace.json
     636 /tmp/output_wo_backtrace.json
```

This enhancement request is to explore when to provide the backtrace output. For example have it in `--format fulljson`, but not have it in `--format json`

**Comment by arlimus** [comment]
:+1: interesting, with `--controls` and that whole `fulljson` piece merged, we are starting to see more of these filter-that-json problems, or also: jq-soluble :grin: 

I'm not sure yet about the correct way...

that `--format json` formatter is still 100% rspec-oriented, with just 2 extra fields that don't hurt. but in the end: `fulljson` is the more complete one. idea: we might opt to rebrand our `json` to `rspecjson` and make sure we create a real minimal `json` as the alternative; something that 100% complements our json profiles without any overhead. we would then leave rspecjson as it is (i.e. vanilla) and just implement a full and minimal json.

**Comment by aaronlippold** [comment]
👍

**Comment by chris-rock** [comment]
solved with #671
